pkgs/tools/text/rpl/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication }:
+
+buildPythonApplication rec {
+  name = "${pname}-${version}";
+  pname = "rpl";
+  version = "1.5.7";
+
+  # Tests not included in pip package.
+  doCheck = false;
+
+  src = fetchFromGitHub {
+    owner="kcoyner";
+    repo="rpl";
+    rev="v${version}";
```

Comment:
Really nitpicky so sorry, but spaces around `=` for these?

Reply:
I've aligned = with vim-lion, hopes that's not a problem xD